lib/banchan/reports/reports.ex

```diff
@@ -250,13 +250,13 @@ defmodule Banchan.Reports do
   @doc """
   File a bug on GitHub
   """
-  def report_bug(%User{} = actor, title, body, actor_url_fn) do
+  def report_bug(%User{} = actor, title, body, bug, actor_url_fn) do
```

Comment:
```suggestion
  def report_bug(%User{} = actor, title, body, bug?, actor_url_fn) do
```

Elixir convention is to name boolean vars with `?` (we make an exception for schema props because it's better for them to map to their db names)